<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>1181</width>
      <height>791</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(85, 85, 255);
selection-background-color: rgb(85, 85, 255);

</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>430</y>
        <width>331</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>156</red>
               <green>39</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>156</red>
               <green>39</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>156</red>
               <green>39</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>85</red>
               <green>85</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="AlternateBase">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>85</red>
               <green>85</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="PlaceholderText">
             <brush brushstyle="NoBrush">
              <color alpha="128">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>156</red>
               <green>39</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>156</red>
               <green>39</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>156</red>
               <green>39</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>85</red>
               <green>85</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="AlternateBase">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>85</red>
               <green>85</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="PlaceholderText">
             <brush brushstyle="NoBrush">
              <color alpha="128">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>156</red>
               <green>39</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>156</red>
               <green>39</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>156</red>
               <green>39</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>85</red>
               <green>85</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="AlternateBase">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>85</red>
               <green>85</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="PlaceholderText">
             <brush brushstyle="NoBrush">
              <color alpha="128">
               <red>255</red>
               <green>235</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
                background-color: #9C27B0; /* Morado */
                border: 2px solid #FFEB3B; /* Amarillo */
                border-radius: 15px;
                color: #FFEB3B; /* Amarillo */
                padding: 15px 32px;
                text-align: center;
                text-decoration: none;
                display: inline-block;
                font-size: 20px;
                font-family: Arial, sans-serif;
                font-weight: bold;
                transition-duration: 0.4s;
                cursor: pointer;
            }
            QPushButton:hover {
                background-color: #7B1FA2; /* Morado oscuro */
                border-color: #FFD600; /* Amarillo claro */
                color: #FFD600; /* Amarillo claro */
            }
            QPushButton:pressed {
                background-color: #7B1FA2; /* Morado oscuro */
                border-color: #FFD600; /* Amarillo claro */
                color: #FFD600; /* Amarillo claro */
                border: 2px solid #FFEB3B; /* Amarillo */
            }
     </string>
         </property>
         <property name="text">
          <string>Jugar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="aboutButton">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
                background-color: #9C27B0; /* Morado */
                border: 2px solid #FFEB3B; /* Amarillo */
                border-radius: 15px;
                color: #FFEB3B; /* Amarillo */
                padding: 15px 32px;
                text-align: center;
                text-decoration: none;
                display: inline-block;
                font-size: 20px;
                font-family: Arial, sans-serif;
                font-weight: bold;
                transition-duration: 0.4s;
                cursor: pointer;
            }
            QPushButton:hover {
                background-color: #7B1FA2; /* Morado oscuro */
                border-color: #FFD600; /* Amarillo claro */
                color: #FFD600; /* Amarillo claro */
            }
            QPushButton:pressed {
                background-color: #7B1FA2; /* Morado oscuro */
                border-color: #FFD600; /* Amarillo claro */
                color: #FFD600; /* Amarillo claro */
                border: 2px solid #FFEB3B; /* Amarillo */
            }</string>
         </property>
         <property name="text">
          <string>Acerca de</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
                background-color: #9C27B0; /* Morado */
                border: 2px solid #FFEB3B; /* Amarillo */
                border-radius: 15px;
                color: #FFEB3B; /* Amarillo */
                padding: 15px 32px;
                text-align: center;
                text-decoration: none;
                display: inline-block;
                font-size: 20px;
                font-family: Arial, sans-serif;
                font-weight: bold;
                transition-duration: 0.4s;
                cursor: pointer;
            }
            QPushButton:hover {
                background-color: #7B1FA2; /* Morado oscuro */
                border-color: #FFD600; /* Amarillo claro */
                color: #FFD600; /* Amarillo claro */
            }
            QPushButton:pressed {
                background-color: #7B1FA2; /* Morado oscuro */
                border-color: #FFD600; /* Amarillo claro */
                color: #FFD600; /* Amarillo claro */
                border: 2px solid #FFEB3B; /* Amarillo */
            }</string>
         </property>
         <property name="text">
          <string>Salir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="backgroundLabel">
      <property name="geometry">
       <rect>
        <x>-24</x>
        <y>20</y>
        <width>1191</width>
        <height>761</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#backgroundLabel {
    background-image: url(:/images/fondomenu.jpg); 
    background-repeat: no-repeat;
    background-position: center;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="logoLabel">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>731</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#logoLabel {
    background-image: url(:/images/logopong.png);
    background-repeat: no-repeat;
    background-position: top center;
    min-height: 10px; 
	border-color: rgb(85, 0, 255);
	
	border-bottom-color: rgb(85, 0, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>backgroundLabel</zorder>
     <zorder>verticalLayoutWidget_2</zorder>
     <zorder>logoLabel</zorder>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>260</y>
        <width>391</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="singlePlayerButton">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
                background-color: #9C27B0; 
                border: 2px solid #FFEB3B; 
                border-radius: 15px;
                color: #FFEB3B; 
                padding: 15px 32px;
                text-align: center;
                text-decoration: none;
                display: inline-block;
                font-size: 20px;
                font-family: Arial, sans-serif;
                font-weight: bold;
                transition-duration: 0.4s;
                cursor: pointer;
            }
            QPushButton:hover {
                background-color: #7B1FA2; 
                border-color: #FFD600; 
                color: #FFD600; 
            }
            QPushButton:pressed {
                background-color: #7B1FA2;
                border-color: #FFD600; 
                color: #FFD600; 
                border: 2px solid #FFEB3B; 
            }
     </string>
         </property>
         <property name="text">
          <string>Un jugador</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="multiPlayerButton">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
                background-color: #9C27B0; 
                border: 2px solid #FFEB3B; 
                border-radius: 15px;
                color: #FFEB3B;
                padding: 15px 32px;
                text-align: center;
                text-decoration: none;
                display: inline-block;
                font-size: 20px;
                font-family: Arial, sans-serif;
                font-weight: bold;
                transition-duration: 0.4s;
                cursor: pointer;
            }
            QPushButton:hover {
                background-color: #7B1FA2; 
                border-color: #FFD600; 
                color: #FFD600; 
            }
            QPushButton:pressed {
                background-color: #7B1FA2; 
                border-color: #FFD600; 
                color: #FFD600; 
                border: 2px solid #FFEB3B; 
            }
     </string>
         </property>
         <property name="text">
          <string>Dos jugadores</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
                background-color: #9C27B0; 
                border: 2px solid #FFEB3B;
                border-radius: 15px;
                color: #FFEB3B; 
                padding: 15px 32px;
                text-align: center;
                text-decoration: none;
                display: inline-block;
                font-size: 20px;
                font-family: Arial, sans-serif;
                font-weight: bold;
                transition-duration: 0.4s;
                cursor: pointer;
            }
            QPushButton:hover {
                background-color: #7B1FA2; 
                border-color: #FFD600; 
                color: #FFD600; 
            }
            QPushButton:pressed {
                background-color: #7B1FA2;
                border-color: #FFD600; 
                color: #FFD600; 
                border: 2px solid #FFEB3B; 
            }
     </string>
         </property>
         <property name="text">
          <string>Atras</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="backgroundLabel_2">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>0</y>
        <width>1201</width>
        <height>781</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#backgroundLabel_2 {
    background-image: url(:/images/fondomenu.jpg); 
    background-repeat: no-repeat;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>backgroundLabel_2</zorder>
     <zorder>verticalLayoutWidget</zorder>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>330</y>
        <width>311</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="returnToMenuButton">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
                background-color: #9C27B0; 
                border: 2px solid #FFEB3B; 
                border-radius: 15px;
                color: #FFEB3B; 
                padding: 15px 32px;
                text-align: center;
                text-decoration: none;
                display: inline-block;
                font-size: 20px;
                font-family: Arial, sans-serif;
                font-weight: bold;
                transition-duration: 0.4s;
                cursor: pointer;
            }
            QPushButton:hover {
                background-color: #7B1FA2; 
                border-color: #FFD600; 
                color: #FFD600; 
            }
            QPushButton:pressed {
                background-color: #7B1FA2;
                border-color: #FFD600; 
                color: #FFD600; 
                border: 2px solid #FFEB3B; 
            }
     </string>
         </property>
         <property name="text">
          <string>Volver al Menú</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="backgroundLabel_3">
      <property name="geometry">
       <rect>
        <x>-470</x>
        <y>-30</y>
        <width>1611</width>
        <height>871</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#backgroundLabel_3 {
    background-image: url(:/images/icono1.png); 
    background-repeat: no-repeat;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="gameover">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>981</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kristen ITC</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
        font-size: 48pt; /* Tamaño de la fuente */
        font-weight: bold; /* Grosor de la fuente */
        color: #FFEB3B; /* Amarillo */
        text-align: center; /* Alineación del texto */
        padding: 20px; /* Relleno alrededor del texto */
        border-radius: 10px; /* Radio de borde para un aspecto redondeado */
        background-color: #7B1FA2;
		text-transform: uppercase;
    }</string>
      </property>
      <property name="text">
       <string>           Game over</string>
      </property>
     </widget>
     <zorder>backgroundLabel_3</zorder>
     <zorder>gridLayoutWidget</zorder>
     <zorder>gameover</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
